scripts/config/pages.js

```diff
@@ -37,6 +37,7 @@ module.exports = [
     ['user/reality_check_summary',              'app/user/reality_check/summary',              'default',    'Reality Check'],
     ['user/reset_passwordws',                   'app/user/reset_password',                     'default',    'Password Reset'],
     ['user/securityws',                         'app/user/security',                           'default',    'Security'],
+    ['user/security/closurews',                 'app/user/security/account_closure',           'default',    'Account Closure'],
```

Comment:
please don't put `ws` in new urls, we used to add them before when marking which pages have moved to websocket, but now we can't change them since other people may have links to them. there's no need to add it for new urls

Reply:
Okay 👍